### PR TITLE
Create rfc3339-validator_ubi_8.4.sh

### PR DESCRIPTION
**Pull request by santosh653**
[root@4ce5bf2f4646 rfc3339-validator]# ./rfc3339-validator_ubi_8.4.sh
Updating Subscription Management repositories.
Unable to read consumer identity

This system is not registered to Red Hat Subscription Management. You can use subscription-manager to register.

Last metadata expiration check: 0:54:11 ago on Mon Sep  6 05:56:02 2021.
Package python36-3.6.8-2.module+el8.1.0+3334+5cb623d7.ppc64le is already installed.
Dependencies resolved.
Nothing to do.
Complete!
Updating Subscription Management repositories.
Unable to read consumer identity

This system is not registered to Red Hat Subscription Management. You can use subscription-manager to register.

Last metadata expiration check: 0:54:13 ago on Mon Sep  6 05:56:02 2021.
Package git-2.27.0-1.el8.ppc64le is already installed.
Dependencies resolved.
Nothing to do.
Complete!
Updating Subscription Management repositories.
Unable to read consumer identity

This system is not registered to Red Hat Subscription Management. You can use subscription-manager to register.

Last metadata expiration check: 0:54:16 ago on Mon Sep  6 05:56:02 2021.
Dependencies resolved.
========================================================================================================================================================================
 Package                           Architecture                         Version                                      Repository                                    Size
========================================================================================================================================================================
Installing:
 wget                              ppc64le                              1.19.5-10.el8                                ubi-8-appstream                              748 k

Transaction Summary
========================================================================================================================================================================
Install  1 Package

Total download size: 748 k
Installed size: 2.9 M
Downloading Packages:
wget-1.19.5-10.el8.ppc64le.rpm                                                                                                          3.3 MB/s | 748 kB     00:00
------------------------------------------------------------------------------------------------------------------------------------------------------------------------
Total                                                                                                                                   3.2 MB/s | 748 kB     00:00
Running transaction check
Transaction check succeeded.
Running transaction test
Transaction test succeeded.
Running transaction
  Preparing        :                                                                                                                                                1/1
  Installing       : wget-1.19.5-10.el8.ppc64le                                                                                                                     1/1
  Running scriptlet: wget-1.19.5-10.el8.ppc64le                                                                                                                     1/1
  Verifying        : wget-1.19.5-10.el8.ppc64le                                                                                                                     1/1
Installed products updated.

Installed:
  wget-1.19.5-10.el8.ppc64le

Complete!
Cloning into 'rfc3339-validator'...
remote: Enumerating objects: 65, done.
remote: Counting objects: 100% (65/65), done.
remote: Compressing objects: 100% (53/53), done.
remote: Total 65 (delta 26), reused 42 (delta 11), pack-reused 0
Unpacking objects: 100% (65/65), 15.60 KiB | 285.00 KiB/s, done.
Already on 'master'
Your branch is up to date with 'origin/master'.
 found to checkout

GLOB sdist-make: /home/rfc3339-validator/rfc3339-validator/setup.py
py36 create: /home/rfc3339-validator/rfc3339-validator/.tox/py36
py36 installdeps: -r/home/rfc3339-validator/rfc3339-validator/requirements_dev.txt
py36 inst: /home/rfc3339-validator/rfc3339-validator/.tox/.tmp/package/1/rfc3339_validator-0.1.4.zip
py36 installed: atomicwrites==1.4.0,attrs==21.2.0,backports.entry-points-selectable==1.1.0,bleach==4.1.0,bump2version==0.5.11,certifi==2021.5.30,charset-normalizer==2.0.4,coverage==4.5.4,distlib==0.3.2,docutils==0.17.1,entrypoints==0.3,filelock==3.0.12,flake8==3.7.8,hypothesis==4.41.3,idna==3.2,importlib-metadata==0.23,importlib-resources==5.2.2,mccabe==0.6.1,more-itertools==8.9.0,packaging==21.0,pkginfo==1.7.1,platformdirs==2.3.0,pluggy==0.13.1,py==1.10.0,pycodestyle==2.5.0,pyflakes==2.1.1,Pygments==2.10.0,pyparsing==2.4.7,pytest==4.6.5,pytest-runner==5.1,readme-renderer==29.0,requests==2.26.0,requests-toolbelt==0.9.1,rfc3339-validator==0.1.4,six==1.16.0,strict-rfc3339==0.7,toml==0.10.2,tox==3.14.0,tqdm==4.62.2,twine==1.14.0,urllib3==1.26.6,virtualenv==20.7.2,wcwidth==0.2.5,webencodings==0.5.1,zipp==3.5.0
py36 run-test-pre: PYTHONHASHSEED='3976244779'
py36 run-test: commands[0] | pip install -U pip
Collecting pip
  Using cached https://files.pythonhosted.org/packages/ca/31/b88ef447d595963c01060998cb329251648acf4a067721b0452c45527eb8/pip-21.2.4-py3-none-any.whl
Installing collected packages: pip
  Found existing installation: pip 19.2.3
    Uninstalling pip-19.2.3:
      Successfully uninstalled pip-19.2.3
Successfully installed pip-21.2.4
py36 run-test: commands[1] | pytest --basetemp=/home/rfc3339-validator/rfc3339-validator/.tox/py36/tmp
========================================================================= test session starts ==========================================================================
platform linux -- Python 3.6.8, pytest-4.6.5, py-1.10.0, pluggy-0.13.1
cachedir: .tox/py36/.pytest_cache
hypothesis profile 'default' -> database=DirectoryBasedExampleDatabase('/home/rfc3339-validator/rfc3339-validator/.hypothesis/examples')
rootdir: /home/rfc3339-validator/rfc3339-validator, inifile: setup.cfg
plugins: hypothesis-4.41.3
collected 3 items

tests/test_rfc3339_validator.py ...                                                                                                                              [100%]

======================================================================= 3 passed in 7.39 seconds =======================================================================
_______________________________________________________________________________ summary ________________________________________________________________________________
  py36: commands succeeded
  congratulations :)
===================================
**Running with version- v0.1.4**

[root@4ce5bf2f4646 rfc3339-validator]# ./rfc3339-validator_ubi_8.4.sh v0.1.4
Updating Subscription Management repositories.
Unable to read consumer identity

This system is not registered to Red Hat Subscription Management. You can use subscription-manager to register.

Last metadata expiration check: 1:46:56 ago on Mon Sep  6 05:56:02 2021.
Package python36-3.6.8-2.module+el8.1.0+3334+5cb623d7.ppc64le is already installed.
Dependencies resolved.
Nothing to do.
Complete!
Updating Subscription Management repositories.
Unable to read consumer identity

This system is not registered to Red Hat Subscription Management. You can use subscription-manager to register.

Last metadata expiration check: 1:46:58 ago on Mon Sep  6 05:56:02 2021.
Package git-2.27.0-1.el8.ppc64le is already installed.
Dependencies resolved.
Nothing to do.
Complete!
Updating Subscription Management repositories.
Unable to read consumer identity

This system is not registered to Red Hat Subscription Management. You can use subscription-manager to register.

Last metadata expiration check: 1:46:59 ago on Mon Sep  6 05:56:02 2021.
Package wget-1.19.5-10.el8.ppc64le is already installed.
Dependencies resolved.
Nothing to do.
Complete!
Cloning into 'rfc3339-validator'...
remote: Enumerating objects: 65, done.
remote: Counting objects: 100% (65/65), done.
remote: Compressing objects: 100% (53/53), done.
remote: Total 65 (delta 26), reused 42 (delta 11), pack-reused 0
Unpacking objects: 100% (65/65), 15.60 KiB | 887.00 KiB/s, done.
Note: switching to 'v0.1.4'.

You are in 'detached HEAD' state. You can look around, make experimental
changes and commit them, and you can discard any commits you make in this
state without impacting any branches by switching back to a branch.

If you want to create a new branch to retain commits you create, you may
do so (now or later) by using -c with the switch command. Example:

  git switch -c <new-branch-name>

Or undo this operation with:

  git switch -

Turn off this advice by setting config variable advice.detachedHead to false

HEAD is now at 5ebeb83 Bump version: 0.1.3 → 0.1.4
 found to checkout
WARNING: Running pip install with root privileges is generally not a good idea. Try `pip3 install --user` instead.
Requirement already satisfied: tox in /usr/local/lib/python3.6/site-packages
Requirement already satisfied: virtualenv!=20.0.0,!=20.0.1,!=20.0.2,!=20.0.3,!=20.0.4,!=20.0.5,!=20.0.6,!=20.0.7,>=16.0.0 in /usr/local/lib/python3.6/site-packages (from tox)
Requirement already satisfied: importlib-metadata>=0.12; python_version < "3.8" in /usr/local/lib/python3.6/site-packages (from tox)
Requirement already satisfied: six>=1.14.0 in /usr/local/lib/python3.6/site-packages (from tox)
Requirement already satisfied: pluggy>=0.12.0 in /usr/local/lib/python3.6/site-packages (from tox)
Requirement already satisfied: packaging>=14 in /usr/local/lib/python3.6/site-packages (from tox)
Requirement already satisfied: filelock>=3.0.0 in /usr/local/lib/python3.6/site-packages (from tox)
Requirement already satisfied: py>=1.4.17 in /usr/local/lib/python3.6/site-packages (from tox)
Requirement already satisfied: toml>=0.9.4 in /usr/local/lib/python3.6/site-packages (from tox)
Requirement already satisfied: platformdirs<3,>=2 in /usr/local/lib/python3.6/site-packages (from virtualenv!=20.0.0,!=20.0.1,!=20.0.2,!=20.0.3,!=20.0.4,!=20.0.5,!=20.0.6,!=20.0.7,>=16.0.0->tox)
Requirement already satisfied: distlib<1,>=0.3.1 in /usr/local/lib/python3.6/site-packages (from virtualenv!=20.0.0,!=20.0.1,!=20.0.2,!=20.0.3,!=20.0.4,!=20.0.5,!=20.0.6,!=20.0.7,>=16.0.0->tox)
Requirement already satisfied: importlib-resources>=1.0; python_version < "3.7" in /usr/local/lib/python3.6/site-packages (from virtualenv!=20.0.0,!=20.0.1,!=20.0.2,!=20.0.3,!=20.0.4,!=20.0.5,!=20.0.6,!=20.0.7,>=16.0.0->tox)
Requirement already satisfied: backports.entry-points-selectable>=1.0.4 in /usr/local/lib/python3.6/site-packages (from virtualenv!=20.0.0,!=20.0.1,!=20.0.2,!=20.0.3,!=20.0.4,!=20.0.5,!=20.0.6,!=20.0.7,>=16.0.0->tox)
Requirement already satisfied: zipp>=0.5 in /usr/local/lib/python3.6/site-packages (from importlib-metadata>=0.12; python_version < "3.8"->tox)
Requirement already satisfied: typing-extensions>=3.6.4; python_version < "3.8" in /usr/local/lib/python3.6/site-packages (from importlib-metadata>=0.12; python_version < "3.8"->tox)
Requirement already satisfied: pyparsing>=2.0.2 in /usr/local/lib/python3.6/site-packages (from packaging>=14->tox)
GLOB sdist-make: /home/rfc3339-validator/rfc3339-validator/setup.py
py36 create: /home/rfc3339-validator/rfc3339-validator/.tox/py36
py36 installdeps: -r/home/rfc3339-validator/rfc3339-validator/requirements_dev.txt
py36 inst: /home/rfc3339-validator/rfc3339-validator/.tox/.tmp/package/1/rfc3339_validator-0.1.4.zip
py36 installed: atomicwrites==1.4.0,attrs==21.2.0,backports.entry-points-selectable==1.1.0,bleach==4.1.0,bump2version==0.5.11,certifi==2021.5.30,charset-normalizer==2.0.4,coverage==4.5.4,distlib==0.3.2,docutils==0.17.1,entrypoints==0.3,filelock==3.0.12,flake8==3.7.8,hypothesis==4.41.3,idna==3.2,importlib-metadata==0.23,importlib-resources==5.2.2,mccabe==0.6.1,more-itertools==8.9.0,packaging==21.0,pkginfo==1.7.1,platformdirs==2.3.0,pluggy==0.13.1,py==1.10.0,pycodestyle==2.5.0,pyflakes==2.1.1,Pygments==2.10.0,pyparsing==2.4.7,pytest==4.6.5,pytest-runner==5.1,readme-renderer==29.0,requests==2.26.0,requests-toolbelt==0.9.1,rfc3339-validator==0.1.4,six==1.16.0,strict-rfc3339==0.7,toml==0.10.2,tox==3.14.0,tqdm==4.62.2,twine==1.14.0,urllib3==1.26.6,virtualenv==20.7.2,wcwidth==0.2.5,webencodings==0.5.1,zipp==3.5.0
py36 run-test-pre: PYTHONHASHSEED='3518177927'
py36 run-test: commands[0] | pip install -U pip
Collecting pip
  Using cached https://files.pythonhosted.org/packages/ca/31/b88ef447d595963c01060998cb329251648acf4a067721b0452c45527eb8/pip-21.2.4-py3-none-any.whl
Installing collected packages: pip
  Found existing installation: pip 19.2.3
    Uninstalling pip-19.2.3:
      Successfully uninstalled pip-19.2.3
Successfully installed pip-21.2.4
py36 run-test: commands[1] | pytest --basetemp=/home/rfc3339-validator/rfc3339-validator/.tox/py36/tmp
========================================================================= test session starts ==========================================================================
platform linux -- Python 3.6.8, pytest-4.6.5, py-1.10.0, pluggy-0.13.1
cachedir: .tox/py36/.pytest_cache
hypothesis profile 'default' -> database=DirectoryBasedExampleDatabase('/home/rfc3339-validator/rfc3339-validator/.hypothesis/examples')
rootdir: /home/rfc3339-validator/rfc3339-validator, inifile: setup.cfg
plugins: hypothesis-4.41.3
collected 3 items

tests/test_rfc3339_validator.py ...                                                                                                                              [100%]

======================================================================= 3 passed in 7.51 seconds =======================================================================
_______________________________________________________________________________ summary ________________________________________________________________________________
  py36: commands succeeded
  congratulations :)

### CODE REVIEW

File: r/rfc3339-validator/rfc3339-validator_ubi_8.4.sh [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+ret=$?
+
+if [ $ret -eq 0 ] ; then
+ echo "$Version found to checkout "
+else
+ echo "$Version not found "
+ exit
+fi
+
+sudo pip3 install tox
```
Comment (seth-priya) [inline r/rfc3339-validator/rfc3339-validator_ubi_8.4.sh:51]: do we need a sudo here?

File: r/rfc3339-validator/rfc3339-validator_ubi_8.4.sh [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+else
+  export VERSION=$1
+fi
+
+if [ -d "rfc3339-validator" ] ; then
+  rm -rf rfc3339-validator
+fi
+
+# Install Dependent S/W
+sudo dnf install -y python36
```
Comment (seth-priya) [inline r/rfc3339-validator/rfc3339-validator_ubi_8.4.sh:30]: we might want to combine these into a single line?

File: r/rfc3339-validator/rfc3339-validator_ubi_8.4.sh [hunk 3]
```diff
@@ -0,0 +1,60 @@
+# ----------------------------------------------------------------------------
+#
+# Package       : rfc3339-validator
+# Version       : v0.1.4
```
Comment (seth-priya) [inline r/rfc3339-validator/rfc3339-validator_ubi_8.4.sh:4]: are these all the version(s) that the script was verified with?

Comment (santosh653) [reply r/rfc3339-validator/rfc3339-validator_ubi_8.4.sh:4]: yes, verified with latest tag version & without passing tag version. Though it works for all the tags.

**Pull request merged**